panda-whale-vekm/dinehard
# Mayson/create tests

Description
This pull request provides the addition of jest test cases that test for the database functionality.

Asana Task
N/A

Testing Instructions
run npm run test to see the test cases run for database.test.js

Additional Notes, Images, etc.
N/A
